scripts/neighbor_advertiser

```diff
@@ -340,7 +340,7 @@ def construct_neighbor_advertiser_slice():
     return slice_obj
 
 
-def wrapped_ferret_request(request_slice, https_endpoint, http_endpoint):
+def wrapped_ferret_request(request_slice, https_endpoint):
     """
         Attempts to reach ferret by first trying HTTPS, failing over to HTTP in
```

Comment:
Can you update this comment to reflect the updated behavior?

Reply:
Good catch! Removed the obsolete comment.